semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/db/DbSemanticConventionUtils.java

```diff
@@ -101,6 +102,49 @@ public static boolean isMongoBackend(Event event) {
     return Lists.newArrayList(Sets.newHashSet(OTEL_DB_OPERATION));
   }
 
+  public static List<String> getAttributeKeysForDbStatement() {
+    return Lists.newArrayList(Sets.newHashSet(OTEL_DB_STATEMENT));
+  }
+
+  public static String getOtelDbOperation(Event event) {
+    String dbOperation =
+        SpanAttributeUtils.getFirstAvailableStringAttribute(
+            event, DbSemanticConventionUtils.getAttributeKeysForDbOperation());
+    String dbStatement =
+        SpanAttributeUtils.getFirstAvailableStringAttribute(
+            event, DbSemanticConventionUtils.getAttributeKeysForDbStatement());
+    if (dbOperation != null) {
```

Comment:
Same as above, return first, we don't need to parse.
And, we can use the same function - `getOperationFromSQLStatement` which trims the string before breaking it.

Reply:
Done